libs/api/utils/src/lib/api-utils.ts

```diff
@@ -26,15 +33,50 @@ export const apiCommonGuard = new Elysia().guard({
     const request = r as ExtendedRequest
     request.requestId = generateRandomIdentifier()
     request.apiKey = apiKey
+    initSentry(request)
 
     const errorCode: ApiErrorCode | undefined = await validateApiKey(apiKey)
     if (errorCode) {
       set.headers['Content-Type'] = 'application/json; charset=utf8'
       throw error(ERROR_LIST[errorCode].code, formatApiErrorResponse(request, errorCode))
+    } else {
+      Sentry.setTag('authorizedKey', 'true')
     }
   },
 })
 
+function initSentry(request: ExtendedRequest) {
+  Sentry.init({
+    dsn: 'https://709a51d30d0a43d222def56984819928@o4508026955038720.ingest.de.sentry.io/4508047143010384',
+    integrations: [
+      Sentry.browserTracingIntegration(),
```

Comment:
**question:** How is this connected to Sentry that is used in `api` that uses Elysia? Can this initialization be shared also for Next.js and other apps or it needs to be defined separately.